Core/Dialog/PSPOskDialog.cpp

```diff
@@ -1173,6 +1178,8 @@ void PSPOskDialog::DoState(PointerWrap &p)
 	if (!s)
 		return;
 
+	// TODO: Should we save currentKeyboard/currentKeyboardLanguage?

```

Comment:
Personally, I don't think so.  It's not exposed to the game in any way, and it might change between versions.  Seems safer not to save.

-[Unknown]